dashboard/app/controllers/projects_controller.rb

```diff
@@ -15,4 +21,16 @@ def template
 
     render template: "projects/#{params[:template]}", layout: nil
   end
+
+  def new
```

Comment:
Fixed in 618308f.  I don't think it's super clean to hardcode strings with level names, but I think you're right that it's better to re-use `levels#show` instead of adding a `projects#new`.